src/plugins/aws/ssh.ts

```diff
@@ -22,11 +22,10 @@ import {
   AwsSshRoleRequest,
 } from "./types";
 
-/** Maximum number of attempts to start an SSH session
- *
- * Each attempt consumes ~ 1 s.
+/**
+ * It can take up to 1 minute for access to propagate on AWS, so set the time limit to 10 minutes.
  */
-const MAX_SSH_RETRIES = 30;
+const TIME_LIMIT_MS = 10 * 60 * 1000;
```

Comment:
```suggestion
const TIME_LIMIT_MS = 60 * 1000;
```

Seems already long enough.

When I analyzed this previously, 99% + of provisioning finished within 8 seconds. Has that changed?